### ISSUE TITLE
PNG exported from GIMP and imported as Sprite2D doesn't show in editor

### ISSUE THREAD

**Issue by 37Rb** [godotengine/godot#74177]
### Godot version

4.0.stable

### System information

OSX Ventura 13.1

### Issue description

I just installed the Godot 4 official release. When I import a sprite from an PNG that I exported from GIMP it doesn't appear in the editor. It does show up when I run the game but not in the editor. I used this same image in Godot 3.5.1 with no problem. I've found that if I re-export it as a new image it works fine so this isn't blocking me at all but as far as I can tell this is a valid PNG image that should work in Godot and it's doesn't. Just reporting in case it helps make the editor more robust.

![The Problem Image](https://user-images.githubusercontent.com/82692355/222236644-53d1eefa-44a6-4bd7-9de5-72159a4b2236.png)

### Steps to reproduce

Import the attached image as a Sprite2D.

### Minimal reproduction project

N/A

**Comment by clayjohn** [comment]
It works fine for me on PopOS 22.04. Maybe this is a macOS specific issue?

![Screenshot from 2023-03-01 10-59-58](https://user-images.githubusercontent.com/16521339/222238419-877eb178-cc72-42dd-bab2-3224bb8b88a2.png)

Could you please provide more information to assist contributors in debugging this issue for you?

**Comment by 37Rb** [comment]
Upon further investigation it seems like the problem had to do with the resource path (and maybe caching?), not the image file itself. Somehow the editor got confused and it wasn't showing. It just so happened that all the other images I tried when troubleshooting it had a different path so they worked. At one point (after I submitted the bug) an error showed that said something about a resource with that path had already been imported. Anyway, I can't reproduce the problem now. Sorry. If it happens again I'll update the issue with more detail.
